src/components/Card/Card.stories.tsx

```diff
@@ -0,0 +1,32 @@
+import type { Meta } from '@storybook/react'
+import React from 'react'
+import { Card } from './index'
+import { CardProps } from './type'
+import { Text } from '../Text'
+
+export default {
+  title: 'Components/Card',
+  component: Card,
+  tags: ['autodocs'],
+  render: () => Card,
+} as Meta<CardProps>
+
+const theme = 'light'
+
+export const CardDefault = (args: CardProps) => (
+  <Card theme={theme} {...args}>
+    <Text color="#212425">Lorem Ipsum</Text>
+  </Card>
+)
+
+export const CardSelect = (args: CardProps) => (
+  <Card {...args} theme={theme} selected>
+    <Text color="#212425">Lorem Ipsum</Text>
+  </Card>
+)
+
+export const CardMaxWidth = (args: CardProps) => (
+  <Card {...args} theme={theme} selected maxWidth="50%">
+    <Text color="#212425">Lorem Ipsum</Text>
+  </Card>
+)
```

Comment:
convert to CSF pattern

- [Storybook Docs](https://storybook.js.org/docs/api/csf#default-export)